cms/README.md

```diff
@@ -0,0 +1,180 @@
+# DOAJ CMS
+
+This directory in the source contains all the content, data and other assets that are managed separately from the
+main software.  This is:
+
+1. **assets** - the static assets, such as images, that are part of the content
+2. **data** - YAML files of structured data to be used in the informational parts of the site
+3. **pages** - a directory structure of markdown files which reflect the structure of the pages on the DOAJ site
+4. **sass** - the source SASS which is compiled into the main site CSS
+
+See below for details for each of these.
+
+## Compiling content
+
+All the markdown and SASS needs to be compiled to HTML and CSS respectively before the site can function.  This can 
+be done in several ways:
+
+### For developers on their local machines
+
+The Markdown and SASS is compiled automatically each time DOAJ restarts, provided you have set `DEBUG=True` in your
```

Comment:
It doesn't auto-reload on changes, no.  The app auto reloads changes to the actual application, but changing the static pages doesn't hit that trigger.  I don't know if that's possible in flask, though we could make our own watcher that handles it if it's necessary.

Reply:
Not sure how necessary it is immediately, but I would probably go with [BrowserSync](https://browsersync.io/) since it’s pretty commonly used for that! It’s pretty nice for front-end development

Reply:
(And has nice testing features like being able to view your local site on different devices)